test/endtoend/operator/operator-latest.yaml

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
nit: is that extra line needed?

Reply:
No not really, must have come while copy-pasting the crds

Reply:
Fixed